DISDRODB/Raw/EPFL/HYMEX_LTE_SOP3/metadata/30.yml

```diff
@@ -9,22 +9,22 @@ source: ''
 source_convention: ''
 source_processing_date: ''
 title: HYMEX_LTE_SOP3
-description: ''
-project_name: HYMEX_LTE_SOP3
+description: HYdrological cycle in the Mediterranean EXperiment (2013)
+project_name: HYdrological cycle in the Mediterranean EXperiment (2013)
 keywords: ''
 summary: ''
 history: ''
-comment: ''
+comment: 'This station ID was assigned to a different station (Les Blaches location) in the previous campaign of this location (HYMEX_LTE_SOP2), in 2012'
 station_id: '30'
-location: ''
-country: ''
-continent: ''
-latitude: -9999
-longitude: -9999
-altitude: -9999
-deployment_status: ''
+location: 'Le Pradel (Grainage) '
```

Comment:
Trailing space at the end ! 
Note that is not needed to put ' ' ... in case ...